variants/aws-dev/Cargo.toml

```diff
@@ -37,6 +37,7 @@ included-packages = [
     "tcpdump",
     "chrony-tools",
     "journald-cloudwatch-logs",
+    "cloudwatch-agent",
 ]
 
```

Comment:
For the lack of a better place to put this comment, you have an extra merge commit, after this commit, you can delete it.